robot_skills/src/robot_skills/ears.py

```diff
@@ -24,7 +24,7 @@ def recognize(self, spec, choices={}, time_out=rospy.Duration(10)):
         if hasattr(self._pre_hook, '__call__'):
             self._pre_hook()
 
-        answer = self._hmi.old_query(spec, choices, timeout=time_out.to_sec())
+        answer = self._hmi.old_query(spec, choices, timeout=time_out.to_sec())  # self._hmi needs to be set in robot
```

Comment:
Is this still used even?

Reply:
I don’t think so. But would mean removal of the entire ears. I would do that in a different PR.

Reply:
:+1: